Jenkinsfile

```diff
@@ -167,8 +181,12 @@ def maybeArchiveArtifacts(params, String suffix) {
             // Archive disk image
             sh label: 'Compress kernel and images', script: """
 rm -fv *.img *.xz kernel*
```

Comment:
Does this rm of *.img throw away memstick.img before you save it below?

Reply:
No, it's put in tarball/ (as the default output root for jenkins-cheribuild) and moved out below